### ISSUE TITLE
Bridge: Bot cue bids but players don't always know what that means

### ISSUE THREAD

**Issue by johnson520** [TricksterCards/TricksterBots#38]
From a customer:
>It [bot] cue bids the opponents suit very often, making a shambles of the bidding, as nobody knows what that means.

**Comment by RussCranberry** [comment]
I agree that this is confusing.  I sometimes wonder if the bot is not sufficiently taking into account the previous bidding by the opponents.
